<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsNewHttpConnectionBase</class>
 <widget class="QDialog" name="QgsNewHttpConnectionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create a New WMS connection</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="Q3GroupBox" name="GroupBox1" >
       <property name="title" >
        <string>Connection Information</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>11</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="TextLabel1_2" >
              <property name="text" >
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel1" >
              <property name="text" >
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel1_3" >
              <property name="text" >
               <string>Proxy Host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel1_3_2" >
              <property name="text" >
               <string>Proxy Port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="txtName" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Name of the new connection</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtUrl" >
              <property name="toolTip" >
               <string>HTTP address of the Web Map Server</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtProxyHost" >
              <property name="toolTip" >
               <string>Name of your HTTP proxy (optional)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtProxyPort" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>7</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip" >
               <string>Port number of your HTTP proxy (optional)</string>
              </property>
              <property name="maxLength" >
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnOk" >
         <property name="text" >
          <string>OK</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel" >
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHelp" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Help</string>
         </property>
         <property name="shortcut" >
          <string>F1</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtUrl</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
